data/kebijakan-privasi.json

```diff
@@ -3,7 +3,7 @@
   "icon":"icon-kebijakan-privasi",
   "link": "/kebijakan-privasi-ketentuan-pengguna/kebijakan-privasi",
   "description": "Kebijakan Privasi ini mengatur mengenai kebijakan dan prosedur dalam mengumpulkan, menggunakan, dan menjaga data yang diperoleh dari pelanggan (selanjutnya disebut <strong class='text-[#212121] dark:text-[#E9E9E9]'>“Anda”</strong> atau <strong class='text-[#212121] dark:text-[#E9E9E9]'>“Pengguna”</strong>) sehubungan dengan penggunaan layanan yang disediakan di aplikasi SAPAWARGA (<strong class='text-[#212121] dark:text-[#E9E9E9]'>“Aplikasi”</strong>). Pada Kebijakan Privasi ini, kecuali secara tegas disebutkan berbeda, berlaku untuk Situs, Aplikasi dan semua produk dan layanan yang disediakan oleh Situs dan Aplikasi.",
-  "updated":"20 September 2022",
+  "updated":"7 November 2022",
```

Comment:
I personally prefer 07 instead, just my subjective opinion to keep the consistancy of the format date 🤣

Reply:
Hello @yoslie , thank you for your opinion. But, I just follow the instruction in clickup. Maybe, you can ask PM for consistancy of format date.